src/ReadLog.jl

```diff
@@ -0,0 +1,134 @@
+# Read LCM log files directly
+
+# export
+#   LCMlog,
+#   close,
+#   readNextEvent,
+#   handle,
+#   subscribe
+
+# also see JuliaLang's RawFD type, but little information found.
+# www.c4learn/c-programming/c-file-structure-and-file-pointer
+mutable struct FILE
```

Comment:
I'm a bit concerned about trying to exactly mirror the `FILE` implementation from C here. Is this actually necessary? The code never uses any of these fields, so I think we could replace all of this with a `Ptr{Void}` field in `lcm_eventlog_t`.